### DISCUSSION TITLE
✗ No Network After 10 Tries  🖧  Check Network Settings

### DISCUSSION THREAD

**Discussion by ucramos** [tteck/Proxmox#3631] [Help Needed] [answered]
I´ve been trying to install a LXC script, and i´ve tried it more than 30 times by now (not kidding).

I´ve tried other scripts with the same message,
```
`    ______     _             __
   / ____/____(_)___ _____ _/ /____
  / /_  / ___/ / __ `/ __ `/ __/ _ \
 / __/ / /  / / /_/ / /_/ / /_/  __/
/_/   /_/  /_/\__, /\__,_/\__/\___/
             /____/

Using Advanced Settings
Using Distribution: debian
Using debian Version: 11
Using Container Type: 0
Using Root Password: Automatic Login
Container ID: 247
Using Hostname: frigate
Using Disk Size: 20
Allocated Cores: 4
Allocated RAM: 4096
Using Bridge: vmbr0
Using IP Address: 192.168.1.247/24
Using Gateway IP Address: 192.168.1.1
Using APT-Cacher IP Address: Default
Disable IPv6: no
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using Vlan: Default
Enable Root SSH Access: no
Enable Verbose Mode: yes
Creating a Frigate LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using local-zfs for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 247 was successfully created.
 ✓ Started LXC Container
 \✗ No Network! ntainer OS   
✗ No Network After 10 Tries
 🖧  Check Network Settings`
```
What i tried until this time:

- Reboot the router
- Reboot Proxmox
- Install with default setting (DHCP)
- Install with Static IP
- Disable IPv6 on Router and LXC config
- Tried different static IPs

Please note that i have all my installed LXCs as Static IP, and all are working fine:
(Some as not running but because i have them stopped, not due to malfunction)

![imagem](https://github.com/user-attachments/assets/580fc85d-82c9-432a-88c0-7bfb92df5b6f)

Also, the static IPs i´ve tried are not being used.
Weird thing is Pi.Alert detect the IP/Machine

![imagem](https://github.com/user-attachments/assets/e8ff5a30-0bba-48a2-817b-25a76fbca5eb)

As i have some Virtual Machines i thought i might exceeded the number of connections of the Bridge but i cannot find any info about that.

Any help would be appreciated.

**Comment by tomekred** [comment]
Hi,

First of all, just want to say that this repo is golden. Thank you for providing it to the community, it makes having fun with homelab so much better, where no one has to spend long(-ish) time to configure LXC etc.


Recently, I got similar issues installing Umami, however it's not exactly 'no network' message (which shows only for a few seconds and disappears like the connection has been made). 

I turned verbose mode on and even though github's ip has been resolved and displayed, the script stops at `git clone -q https://github.com/umami-software/umami.git /opt/umami` saying it can't resolve github.com. 

Until now everything has been working correctly, I installed gitea about a week ago without any issues. I tried another script (homarr) and get the same result (can't clone git repo).
I ran ping on a debian vm and in proxmox shell and both works, so it seems to be odd that I can't connect.

One thing that comes to my mind is that I recently installed nginx proxy manager so I can access one of the docker images using public domain. Could this generate some problems? In general, I don't experience any issues with any other devices and apps.

It seems like my issue is not the one that OP has, but maybe it is somehow relevant?

**Comment by tteck** [comment, marked as answer]
Both are network issues, not script issues.

**Reply by ucramos** [reply]
Sorry if it appeared as I was "blaming" the script for this. Not my intention. Just trying to get some feedback from others and how they solved this problem.
